python/spec/dependabot/python/file_parser_spec.rb

```diff
@@ -1212,18 +1212,18 @@
       end
     end
 
-    context "with a pyproject.toml and pyproject.lock" do
+    context "with a pyproject.toml in poetry format and pyproject.lock legacy poetry lock file" do
       let(:files) { [pyproject, pyproject_lock] }
       let(:pyproject) do
         Dependabot::DependencyFile.new(
           name: "pyproject.toml",
-          content: fixture("pyproject_files", "pyproject.toml")
+          content: fixture("pyproject_files", "poetry.toml")
```

Comment:
Does the file name matter here? It's looks a little strange to load content from `poetry.toml` here but have the DependacyFile name `pyproject.toml` in this test and a few others.